ste/mgr-JobMgr.go

```diff
@@ -571,13 +571,25 @@ func (jm *jobMgr) reportJobPartDoneHandler() {
 
 	for {
 		select {
-		case <-jm.reportCancelCh:
+		case _, ok := <-jm.reportCancelCh:
+			if ok {
+				jm.Panic(fmt.Errorf("Received message on reportCancelCh instead of close signal"))
+			}
+			jpm, ok := jm.jobPartMgrs.Get(0)
+			if ok {
+				plan := jpm.Plan()
+				if plan.JobStatus() == common.EJobStatus.InProgress() ||
+					plan.JobStatus() == common.EJobStatus.Cancelling() {
+					jm.Panic(fmt.Errorf("reportCancelCh received close, job still in progress"))
+				}
+			} else {
+				jm.Log(pipeline.LogError, "Part(0) of job invalid")
```

Comment:
nit, log jobId like L595

Reply:
Yes it will go inside job log, so we don't need it.